.ddev/docker-compose.elasticvue.yaml

```diff
@@ -0,0 +1,16 @@
+services:
+  elasticvue:
+    image: "cars10/elasticvue:1.0.4"
+    #command: /docker-entrypoint.sh nginx -g 'daemon off;'
+    container_name: ddev-${DDEV_SITENAME}-elasticvue
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+      com.ddev.approot: ${DDEV_APPROOT}
+    environment:
+     - 'ELASTICVUE_CLUSTERS=[{"name": "dev cluster", "uri": "http://${DDEV_SITENAME}.ddev.site:9200/", "password": "", "user": ""}]'
```

Comment:
This way the elasticsearch url will be available as a predefined cluster:

![image](https://github.com/wunderio/next-drupal-starterkit/assets/185412/74307c18-6729-4648-8125-560b40d9e046)

Handy!